File: Java/0832_Flipping_an_Image.java
class Solution {
    public int[][] flipAndInvertImage(int[][] image) {
        int n = image.length;
        int[][] result = new int[n][n];
        
        for(int i = 0; i < n; i++){
            for(int j = n-1; j >= 0; j--){
                result[i][n-1-j] = image[i][j] == 0? 1: 0;
            }
        }
        return result;
    }
}

File: CS/0832_Flipping_an_Image.cs
public class Solution {
    public int[][] FlipAndInvertImage(int[][] image) {
        int n = image.Length;
        int[][] result = new int[n][];
        
        for(int i = 0; i < n; i++){
            result[i] = new int[n];
            for(int j = 0; j < n; j++){
                result[i][n-1-j] = (image[i][j] + 1) % 2;
            }
        }
        return result;
    }
}
